### PR TITLE
Add --verify-committee-file command

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: stakewise_cli/commands/verify_committee_file.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    # Decrypt the data with the AES session key
+    cipher_aes = AES.new(session_key, AES.MODE_EAX, nonce)
+    try:
+        private_keys = cipher_aes.decrypt_and_verify(ciphertext, tag)
+    except (UnboundLocalError, ValueError, TypeError):
+        print("Verify failed")
+
+    message = b"\xab" * 32
+    public_keys = [G2ProofOfPossession.SkToPk(key) for key in private_keys]
+    signatures = [G2ProofOfPossession.Sign(key, message) for key in private_keys]
```
Comment (tsudmi) [inline stakewise_cli/commands/verify_committee_file.py:59]:
```suggestion
    agg_public_key = G2ProofOfPossession._AggregatePKs(public_keys)
```

File: stakewise_cli/commands/verify_committee_file.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    agg_sig = G2ProofOfPossession.Aggregate(signatures)
+    assert G2ProofOfPossession.FastAggregateVerify(public_keys, message, agg_sig)
+
+    click.secho(
+        "Submit the post to https://forum.stakewise.io with the following aggregated signature:",
```
Comment (tsudmi) [inline stakewise_cli/commands/verify_committee_file.py:64]:
```suggestion
        "Please submit the following post to https://forum.stakewise.io for the operator who sent the shard file:",
```

File: stakewise_cli/commands/verify_committee_file.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        bold=True,
+        fg="green",
+    )
+    click.echo(agg_sig)
```
Comment (tsudmi) [inline stakewise_cli/commands/verify_committee_file.py:68]:
```suggestion
    click.echo(f"Shard Aggregated Public Key: {Web3.toHex(agg_public_key)}")
```

**Comment by tsudmi** [comment]
Pushed create-shard-pubkeys and verify-shard-pubkeys commands, @unxnn please verify

**Comment by unxnn** [comment]
> Pushed create-shard-pubkeys and verify-shard-pubkeys commands, @unxnn please verify

Thank you @tsudmi, checking.

**Comment by unxnn** [comment]
> Pushed create-shard-pubkeys and verify-shard-pubkeys commands, @unxnn please verify

@tsudmi Tested, seems to work

**Pull request merged**